emailtemplateservice/tool/src/main/webapp/WEB-INF/templates/modify_email.html

```
@@ -1,15 +1,15 @@
-    <head th:replace="fragments/common :: head"/>
+    <head th:replace="~{fragments/common :: head}"/>
     <head>
         <title th:text="#{modifyemail.page.title}">Modify Email Template</title>
     </head>
 
     <body>
         <div class="portletBody container-fluid">
-            <div th:if="${emailTemplate.id == null}" id="menu" th:include="fragments/menus :: main (new)" />
-            <div th:unless="${emailTemplate.id == null}"id="menu" th:include="fragments/menus :: main (edit)" />
+            <div th:if="${emailTemplate.id == null}" id="menu" th:insert="~{fragments/menus :: main (new)}" />
+            <div th:unless="${emailTemplate.id == null}"id="menu" th:insert="~{fragments/menus :: main (edit)}" />
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)